packages/cli/src/commands/auth/CommonAuthArgs.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { CliCommonArgs } from "../../CliCommonArgs.js";
+
+export interface CommonAuthArgs extends CliCommonArgs {
+  baseUrl: string;
```

Comment:
We should probably be more consistent here with baseUrl vs stack

Reply:
I borrowed this from the `upload site` command but agreed